tools/cli/commands/changelog.js

```diff
@@ -308,6 +308,16 @@ async function changelogAdd( argv ) {
 
 		const promptType = await changelogAddPrompt( argv, needChangelog, uniqueProjects );
 
+		// Bail if user doesn't want to auto-add.
+		if ( ! promptType.autoPrompt ) {
```

Comment:
Doesn't quite work.
```
Checking if changelog files are needed. Just a sec...
? Found 2 projects that can accept the same changelog file.
  Found 1 project(s) that requires manual configuration. 
  Add same changelog file to 2 project(s)? Yes
Auto changelog cancelled. You can run 'jetpack changelog add [project-type/project]' to add changelogs individually.
```
Looks like this should fix it
```suggestion
		if ( ! promptType.autoAdd && ! promptType.autoPrompt ) {
```

Reply:
Ah ha, ty for the catch.